plugins/coreboot/coreboot.quirk

```diff
@@ -0,0 +1,2 @@
+[SmbiosManufacturer=LENOVO]
+VersionFormat = lenovo-thinkpad-apci-prefixed
```

Comment:
@hughsie  do you think this has any unexpected effects outside of the coreboot plugin?

Reply:
I think calling it VersionFormat might be asking for trouble. Perhaps `VersionPrefix`?

Reply:
Specifically that version format that is used isn't defined.

Would this work as a custom `Flags` key instead?  I think then it would be more logical to me.

Reply:
And if it doesn't work as a custom `Flags` key I think this should be a key that is coreboot specific (IE something like `CorebootVersionQuirks = lenovo-cbet-prefix`)

Reply:
I wasn't aware that quirks are shared across plugins.
Can I simply add custom key value pairs like this?:
> [SmbiosManufacturer=LENOVO]
> CorebootVersionQuirks = lenovo-cbet-prefix

Reply:
Yup, that works, thanks.